src/index.html

```diff
@@ -9,21 +9,37 @@
     <link href="output.css" rel="stylesheet">
   </head>
   <body class="bg-darkBlue">
-    <header>
-      <div class="flex items-center justify-between px-12 sm:px-7 py-12 max-w-[1640px] m-auto">
+    <header class="h-20 sm:h-32 absolute w-full">
+      <div class="flex items-center justify-between sm:px-7 p-5 sm:py-12 max-w-[1640px] m-auto">
         <div class="min-w-0">
           <a href="#"
             class="text-4xl font-bold leading-7 text-gray-900 sm:truncate sm:tracking-tight">coinix</a>
         </div>
         <div class="flex items-center gap-x-6">
           <a href="#about-us" class="hidden sm:block text-xl font-normal text-[#fff] hover:text-[#7ab9ed]">About Us</a>
-          <a href="#approach" class="hidden sm:block text-xl font-normal text-[#fff] ml-3 hover:text-[#7ab9ed]">Approach</a>
-          <a href="#solutions" class="hidden sm:block text-xl font-normal text-[#fff] ml-3 hover:text-[#7ab9ed]">Solutions</a>
+          <a href="#approach"
+            class="hidden sm:block text-xl font-normal text-[#fff] ml-3 hover:text-[#7ab9ed]">Approach</a>
+          <a href="#solutions"
+            class="hidden sm:block text-xl font-normal text-[#fff] ml-3 hover:text-[#7ab9ed]">Solutions</a>
           <a href="#contact" class="text-base sm:text-xl font-normal text-[#fff] ml-3 hover:text-[#7ab9ed]">Contact</a>
         </div>
       </div>
     </header>
-    
+    <main id="main">
+      <section class="flex justify-start items-center h-screen bg-gray-900 py-24 sm:py-36">
+        <div class="mx-auto w-full max-w-[1640px] px-5 sm:px-7">
```

Comment:
Let’s replace `mx-auto w-full max-w-[1640px] px-5 sm:px-7` with `container lg:px-8`.
This will be a standard container. With horizontal spacing.
For now it is smaller than yours, but we can change it in the tailwind config if needed.
so that the content width is the same everywhere.